tests/test_getRaw.py

```diff
@@ -68,17 +67,17 @@ def testRaw(self):
         self.assertEqual(raw.getDetector().getName(), "R:0,3 S:0,1")
         origin = raw.getWcs().getSkyOrigin()
         self.assertAlmostEqual(
-            origin.getLongitude().asDegrees(), 0.005865, 6)
+            origin.getLongitude().asDegrees(), 0.0058520, 6)
```

Comment:
Why was this value changed?  Same question below.

Reply:
The basic problem is that old phosim data had incorrect WCS information in the FITS headers: the date of equinox was not J2000 but was, instead, the date of observation. The old technique for fixing this isn't easily applicable with the new code, and was slightly incorrect. The new technique is more accurate. (Not that the difference between the techniques really matters, since the WCS from phosim isn't meant to be highly accurate in the first place. The important thing is to fix the gross error).